test/framework/kogitojobsservice.go

```diff
@@ -26,10 +26,8 @@ import (
 )
 
 // InstallKogitoJobsService install the Kogito Jobs Service component
-func InstallKogitoJobsService(namespace string, installerType InstallerType, replicas int, persistence, events bool) error {
-	resource := newKogitoJobsServiceResource(namespace, replicas)
-	setKogitoJobsServicePersistence(resource, persistence)
-	return InstallService(&KogitoServiceHolder{KogitoService: resource}, installerType, "jobs-service", GetCliFlags(persistence, events))
+func InstallKogitoJobsService(namespace string, installerType InstallerType, jobsService *KogitoServiceHolder) error {
```

Comment:
`namespace` not needed here as it is already defined in the `KogitoServiceHolder` I guess.